main/acle.md

```diff
@@ -2830,6 +2826,47 @@ The following table lists the architectures feature mapping for AArch64
    | 580           | `FEAT_SME2`              | sme2          | ```ID_AA64PFR1_EL1.SMEver >= 0b0001```    |
    | 650           | `FEAT_MOPS`              | mops          | ```ID_AA64ISAR2_EL1.MOPS >= 0b0001```     |
 
+### Dependencies
+
+If a feature depends on another feature as defined by the table below then
+no need to explicitly state in the attribute. These dependencies are taken
```

Comment:
I think this should be a little more explicit that both:
*  the depended-on feature _need not_ be specified
*  the depended-on feature _may_ be specified